crates/gitbutler-branch-actions/src/move_commits.rs

```diff
@@ -32,7 +33,7 @@ pub(crate) fn move_commit(
 
     let (ref mut source_branch, source_status) = applied_statuses
         .iter_mut()
-        .find(|(b, _)| b.head == commit_id)
+        .find(|(b, _)| b.id == source_branch_id)
         .ok_or_else(|| anyhow!("commit {commit_id} to be moved could not be found"))?;
```

Comment:
Let's update this error in order to convey that actually the source branch is the thing that could not be found